apps/lobe-chat/1.37.1/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.37.1
+    image: lobehub/lobe-chat:v1.37.2
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
The only difference between these two lines of code is the update to the Docker image version from `v1.37.1` to `v1.37.2`. There are no irregularities, potential issues, or immediate optimization suggestions based solely on this change.